docs/cow-protocol/concepts/how-it-works/how-intents-are-formed.mdx

```
@@ -0,0 +1,318 @@
+beforeAllFees.buyAmount = quote.buyAmount\
+```
+
+## Protocol Fee
+
+The protocol fee is expressed as basis points (`protocolFeeBps`) in the quote response.
+
+### Sell orders (buy token units)
+
+For sell orders, `quote.buyAmount` is already **after** the protocol fee has been deducted:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)